bin/helpers/functions.sh

```diff
@@ -4,6 +4,6 @@
 function get_linker_ldflags {
     echo "
         -X 'github.com/mysterium/node/server.mysteriumApiUrl=${MYSTERIUM_API_URL}'
```

Comment:
Not the same with `mysteriumApiUrl`?

Reply:
No, that was was not renamed in code yet.